### ISSUE TITLE
Unify pro - Smartaudio + microphone audio

### ISSUE THREAD

**Issue by FPVlog** [ImpulseRC/OSD#5]
Hi

I read the manual and couldn’t find anywhere if it is possible to wire the Unify so it uses both smartaudio and audio from the microphone. 

Is it possible at all? Should I jump the two pads and see if it works?

Cheers

**Comment by awolf78** [comment]
Yes, jump the two pads. Solder the SmartAudio wire to the VTX TX pad on the WolfPDB. Then solder a small wire from VTX TX to the Audio pad.

In rare cases this can disrupt SmartAudio communication - then you might have to make a choice between Audio and SmartAudo. Not much we can do about it, that is a design issue of the Unify VTX. TBS recently came out with a Unify version with separate Audio and SmartAudio wire...

**Comment by mortenb1337** [comment]
I had the same question, and came here. Did it work for you?

**Comment by greezmen** [comment]
any plans for smart audio 2.1 support? or iam doing something wrong  its nit working with my hv pro race v2 unify thank you

**Comment by awolf78** [comment]
SA V2.1 already implemented but not released yet, I can get you a beta if you like.

**Comment by question2019** [comment]
hi, pls can u send me the link to the beta with SmartAudio 2.1 Support?
Thanks!

**Comment by redoakleaves** [comment]
It is supported by the new firmware-version which is out since a couple of days, i guess. You should be able to update it right now.

**Comment by question2019** [comment]
works perfect !!! thanks

**Comment by TinoFpv** [comment]
Hi guys, noob here needs help.
I got a foxeer predator v4 on a tbs unify pro v3, the impulse ods have from left to right VTX control, audio out, so I have to bridge them?But when I do so i can't here the audio.
What to do?
Thanks and sorry for the noob question, first build.

**Comment by awolf78** [comment]
I assume you are using the Apex OSD. Easiest is to strip the SmartAudio wire from the Unify a bit longer and solder it across both the VTX control and Audio pad. Please check that you have the volume turned up on your goggles. Are you sure the audio on your goggles is working?

**Comment by TinoFpv** [comment]
Thanks awolf78, it was the jack not going right in.
I soldered now vtx and audio together on the apex osd.It works well.
Last question is what to do with the violet cable coming out from the foxeer.
Don’t now where to solder it :/
Many thanks for help.

**Comment by awolf78** [comment]
The Violett (=Purple in Englisch) is the current sensor I guess. That one is not necessary. Make sure the VTX ground and camera ground are both soldered to the common ground pad.

**Comment by TinoFpv** [comment]
Purple, exactly!All ground cables are fine.Apex osd is showing the right leds, so I think It is ok.
Thanks again awolf!

**Comment by MrMPod** [comment]
> Apex OSD. Easiest is to strip the SmartAudio wire from the Unify a bit longer and solder it across both the VTX control and Audio pad.

but am I right that to change my vtx chanels I need PC computer and ImpulseRC OSD application ?
or is there way to change my vtx chanels and power without computer ?

**Comment by awolf78** [comment]
No, you can change those in the OSD menu. Read [here](https://github.com/ImpulseRC/OSD/wiki/Firmware:-Video-Transmitter)

**Comment by MrMPod** [comment]
so is it necessary strip  the SmartAudio wire or just solder bridged the two pads VTX Control to the Audio Out pad on APEX OSD (I have already connected audio to my Unify VTX)

**Comment by awolf78** [comment]
You can strip the wire longer or bridge the two pads - whatever is easier.

**Comment by MrMPod** [comment]
you are right - strip the SmartAudio wire is easier than bridged the two pads :)
and it is working great, thank you very much for help
